spec/index.html

```diff
@@ -46,7 +46,8 @@
         companyURL: "https://digitalbazaar.com/" },
       { name:       "Gregg Kellogg",
         url:        "https://greggkellogg.net/",
-        w3cid:      "44770" }
+        w3cid:      "44770" },
+      { name:       "Dan Yamamoto" }
```

Comment:
```suggestion
      { name:       "Dan Yamamoto" 
        w3cid:       139477}
```

Reply:
Missing comma, but thanks.

Reply:
Thank you @iherman for adding my w3cid.

It seems to me that I should not add my company info here because my employer is currently not a W3C member.
Should I add my github URL "https://github.com/yamdan" here? That is my only "personal" URL...

Reply:
We can use your GitHub url,p.

Reply:
Indeed, not your company info. The github URL should be o.k.

Reply:
Thanks, then it would look something like this:
```suggestion
      { name:       "Dan Yamamoto",
        url:        "https://github.com/yamdan",
        w3cid:      "139477" }
```